src/coreclr/interpreter/CMakeLists.txt

```diff
@@ -5,9 +5,18 @@ set(INTERPRETER_SOURCES
   intops.cpp
   eeinterp.cpp)
 
+# linking against gcinfo causes failures on x86
```

Comment:
It would be better to not to include the interpreter directory in the parent CMakeLists.txt when the FEATURE_INTERPRETER is not enabled.